Datastore/src/Connection/ConnectionInterface.php

```diff
@@ -20,6 +20,8 @@
 /**
  * Represents a connection to
  * [Datastore](https://cloud.google.com/datastore/).
+ *
+ * @internal
```

Comment:
I added `@internal` to check whether this passes the BC compat checks. I can remove them at all places if these are unwanted.